recipes/eleuther_eval.py

```diff
@@ -316,6 +316,11 @@ def __init__(
         self._dtype = dtype
         self._enable_kv_cache = enable_kv_cache
 
+        if hasattr(self._tokenizer, "pad_id") and self._tokenizer.pad_id is not None:
```

Comment:
We really need to an an abstract class for our tokenizers, but I think it's safe to assume in the meantime that tokenizer.pad_id exists for each tokenizer as our recipes use this. I think we can skip this hasattr check.